INSTALL.md

```diff
@@ -539,13 +539,16 @@ email will be disabled.
 
 The easiest way to create a new user is to do so from a client like [Element](https://element.io/).
 
-Alternatively you can do so from the command line if you have installed via pip.
+Alternatively, you can do so from the command line if you have installed synapse via package.
 
 This can be done as follows:
```

Comment:
This doesn't really make much sense as it is. 

```suggestion
Alternatively, you can do so from the command line. This can be done as follows:
```